pkg/bucketeer/model/get_evaluation_request.go

```diff
@@ -1,19 +1,24 @@
 package model
 
-import "github.com/bucketeer-io/go-server-sdk/pkg/bucketeer/user"
+import (
+	"github.com/bucketeer-io/go-server-sdk/pkg/bucketeer/user"
+	"github.com/bucketeer-io/go-server-sdk/pkg/bucketeer/version"
+)
 
 type GetEvaluationRequest struct {
-	Tag       string       `json:"tag,omitempty"`
-	User      *user.User   `json:"user,omitempty"`
-	FeatureID string       `json:"featureId,omitempty"`
-	SourceID  SourceIDType `json:"sourceId,omitempty"`
+	Tag        string       `json:"tag,omitempty"`
+	User       *user.User   `json:"user,omitempty"`
+	FeatureID  string       `json:"featureId,omitempty"`
+	SourceID   SourceIDType `json:"sourceId,omitempty"`
+	SDKVersion string       `json:"sdkVersion,omitempty"`
 }
 
 func NewGetEvaluationRequest(tag, featureID string, user *user.User) *GetEvaluationRequest {
```

Comment:
@kakcy, can you write a test to ensure the values are inserted correctly when creating the `GetEvaluationRequest`?
Same for the [NewRegisterEventsRequest](https://github.com/bucketeer-io/go-server-sdk/pull/118/files#diff-354b7f6da510b7db9b0ad8be542f9275a1a328eb710070fbacf53c7ef20062d5R11)

Reply:
`TestNewGetEvaluationRequest` already existed, so I made some changes.
I added `TestNewRegisterEventsRequest`.
https://github.com/bucketeer-io/go-server-sdk/pull/118/commits/5e9281e3c1ae6cd1b00730ea6028578e6dc9f040